src/firefly/js/templates/hydra/HydraViewer.jsx

```diff
@@ -135,3 +165,46 @@ function showExpandedView ({expanded}) {
 function closeExpanded() {
     dispatchSetLayoutMode(LO_MODE.expanded, LO_VIEW.none);
 }
+
+
+function makeMenuItems(groups, allSearchItems) {
+    const menuItems=[];
+    if (groups.length > 0) {
+        groups.forEach((g) => {
+            const category = g.title;
+            Object.entries(g.searchItems).forEach(([k,v]) => {
+                menuItems.push({label: k, action:k, primary: !!v?.primary, category, clickHandler: searchClickHandler});
+            });
+        });
+    } else {
+        Object.entries(allSearchItems).forEach(([k,v]) => {
+            menuItems.push({label: k, action:k, primary: !!v?.primary, clickHandler: searchClickHandler});
+        });
+    }
+    return menuItems;
+}
+
+function HydraLanding({icon, title, desc, slotProps, ...props} ) {
+
+    const Greetings = () => (
+        <Stacker startDecorator={icon} direction='column' alignItems='start'>
+            <Typography level='h4'>{title || 'Welcome message here'}</Typography>
```

Comment:
If we add appTitle this can use "Welcome to ${appTitle}" as fallback (how Default Landing Page does it)